archive/common/src/main/scala/uk/ac/wellcome/platform/archive/common/progress/models/Callback.scala

```diff
@@ -18,27 +18,19 @@ case object Callback extends URIConverters {
     callbackUri.map(Callback(_))
   }
 
-  val pendingString = "pending"
-  val succeededString = "succeeded"
-  val failedString = "failed"
+  val processingString = "processing"
+  val successString = "success"
+  val failureString = "failure"
 
   case object Pending extends CallbackStatus {
-    override def toString: String = pendingString
+    override def toString: String = processingString
   }
 
   case object Succeeded extends CallbackStatus {
-    override def toString: String = succeededString
+    override def toString: String = successString
```

Comment:
As with `pending`, I think it's a bit confusing that the string doesn't match the status.

Reply:
I didn't rename them out of laziness to be honest, because that's a refactoring change that affects all apps potentially. But one nice side effect of not renaming the scala object is that the stuff that it's currently stored in dynamo is still readable because scanamo serialises using the object name, not the toString method.
I'm tempted to leave them as is, I'm not sure the strings should explicitly match the object name given that they are more external serialisation than anything else.